flojoy

```diff
@@ -329,28 +329,28 @@ echo ''
 # Initializing FLOJOY-WATCH RQ Worker
 cd $CWD
 
-npx ttab -t 'Flojoy-watch RQ Worker' "$venvCmd; export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES; rq worker flojoy-watch"
+# npx ttab -t 'Flojoy-watch RQ Worker' "$venvCmd; export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES; rq worker flojoy-watch"
```

Comment:
we can get rid of all these commented out lines

Reply:
I think we can get rid of the old script entirely, right?

Reply:
yes I think so